examples/styling.js

```diff
@@ -3,6 +3,14 @@
 /*
   <html>
     <style>
+      #waveform ::part(wrapper) {
+        --box-size: 10px;
+        background-image: 
+          linear-gradient(transparent calc(var(--box-size) - 1px), blue var(--box-size), transparent var(--box-size)), 
+          linear-gradient(90deg, transparent calc(var(--box-size) - 1px), blue var(--box-size), transparent var(--box-size));
+        background-size: 100% var(--box-size), var(--box-size) 100%;
+      }
+
```

Comment:
This example doesn't really show what this can achieve. To really see it I will need to add zoom, but what is your opinion I should do here?

Reply:
Sure, let’s add minPxPerSec: 100